app/src/lib/git/stash.ts

```diff
@@ -0,0 +1,7 @@
+export interface IStashEntry {
+  /** The name of the branch at the time the entry was created. */
+  readonly branchName: string
+
+  /** The SHA of the commit object created as a result of stashing. */
+  readonly sha: string
```

Comment:
💭 could this be something more descriptive than `sha`?

Reply:
Changed - https://github.com/desktop/desktop/pull/6948/commits/01b9731821e6da77b4e2bd3d3c873cd3c32fe819